pages/Documents/ConversationalAI/Third-PartyBots/e-bot7/basic-content.md

```diff
@@ -0,0 +1,110 @@
+---
+pagename: Basic Content
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+subfoldername: e-bot7
+permalink: third-party-bots-e-bot7-basic-content.html
+indicator:
+---
+
+### The Welcome Event
+
+The behaviour of the welcome event is different depending on whether the bot is for chat and messaging. This divergence comes down to the way that each individual Liveperson product works.
+
+#### Messaging Conversations
+
+A Messaging conversation qualifies as "initiated" from a Conversational Cloud perspective only after the consumer sends
+their first message. The consumer is prompted for their initial message in the channel they have chosen to initiate the
+conversation. As a result, the consumer’s first message will be the entry point for the main flow in e-bot7 which can be processed by a `Visitor Input` node.
+In this scenario, all bot messages in the flow which are created before this node will be skipped.
+
+<img class="fancyimage" style="width:450px" src="img/ThirdPartyBots/e-bot7-messaging-flow.png">
+<figure>Figure 2.1 Example conversation flow for messaging without support of the `Welcome` event</figure>
+<br>
+
+#### Chat Conversations
+
+A Chat conversation is considered started when the chat is routed to an agent. Best practice is for the agent to provide the first response.
+In this scenario, all bot messages in the flow which are created before the first `Visitor Input` node will be processed.
+Make sure to have `Bot Message` nodes after the `Start Flow` entrypoint to utilise the `Welcome` event.
+
+<img class="fancyimage" style="width:450px" src="img/ThirdPartyBots/e-bot7-chat-flow.png">
+<figure>Figure 2.2 Example conversation flow for chats with support of the `Welcome` event</figure>
+<br>
+
+### Bot Messages
+
+The flow of a bot in e-bot7 can include three different types of messages a bot can send. These are `Bot Message: Text`, `Bot Message: Image`, `Bot Message: Video`. All three types are supported with some restrictions:
```

Comment:
Maybe add all of these types in the table and make a column `Restrictions` that contains  if any restriction applies to the given type